examples/paywall-tester/src/main/java/com/revenuecat/paywallstester/ui/screens/main/paywalls/PaywallsScreen.kt

```diff
@@ -45,6 +49,17 @@ fun PaywallsScreen(
     LazyColumn {
         items(SamplePaywalls.SampleTemplate.values()) { template ->
             val offering = samplePaywallsLoader.offeringForTemplate(template)
+            val myAppPurchaseLogic = MyAppPurchaseLogic(
+                performPurchase = { _, _ ->
+                    println("Hello from performPurchase!")
```

Comment:
Should we remove these? If we want to keep logs, we should use `Log.d` instead, so it's better formatted with logcat.